packages/webview_flutter/webview_flutter_wkwebview/example/ios/RunnerTests/FWFWebViewFlutterWKWebViewExternalAPITests.m

```diff
@@ -0,0 +1,28 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
```

Comment:
Where are the project changes to add this file?

Reply:
The project changes didn't seem like they were related to adding the file. I could be wrong though I went ahead and added a commit with the project changes from building the example app:

https://github.com/flutter/plugins/pull/7071/commits/b9aa957764c07d3d3ba3335d36105042abacdda6

The file was added to the `webview-umbrella.h` file and its included in `@import webview_flutter_wkwebvew;` for the test.

Reply:
Those are indeed not related; they are auto-migrations from `flutter`. You're missing project changes to add this test file; it's not part of the project, and so this test [isn't being run](https://logs.chromium.org/logs/flutter/buildbucket/cr-buildbucket/8789519639335489425/+/u/Run_plugin_tests/native_test/stdout).

> The file was added to the `webview-umbrella.h` file and its included in `@import webview_flutter_wkwebvew;` for the test.

That's the new header; the problem is the test file itself (which isn't part of the plugin, so isn't covered by the podspec).

Reply:
Ah, good catch. Now I understand. I didn't realize this was the `Tests` file. I added the file in Xcode, so the project file should be updated properly now.